src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -384,6 +387,10 @@ private function getDefinitionSchema(string $resourceClass, ResourceMetadata $re
             $normalizedPropertyName = $this->nameConverter ? $this->nameConverter->normalize($propertyName) : $propertyName;
 
             if ($propertyMetadata->isRequired()) {
+                if (false === $propertyMetadata->isWritable()) {
+                    throw new RuntimeException(sprintf('The property "%s" of the resource "%s" can not be required and read-only at the same time.', $propertyName, $resourceClass));
```

Comment:
IMO this loader should be fixed and not set required to true if the property isn't writeable. WDYT?